config/owasp/suppressions.xml

```diff
@@ -38,9 +33,9 @@
     </notes>
     <cve>CVE-2020-8908</cve>
   </suppress>
-  <suppress>
-    <notes>jackson.core/jackson-databind@2.15.0</notes>
-    <cve>CVE-2023-35116</cve>
+  <suppress until="2023-12-10">
```

Comment:
Suppress for a month ?

Reply:
The CVE comes from a package from the latest version of Spring boot. I'm fairly confident that they will have a patch release out soon which will make it go away. There is a patch version of the package available already but bumping just that package breaks the build.